src/Playground.iOS/Playground_iOSViewController.cs

```diff
@@ -21,16 +21,16 @@ public Playground_iOSViewController () : base ("Playground_iOSViewController", n
         {
 
             //Here we accept any certificate and just print the cert's data.
-            ServicePointManager.ServerCertificateValidationCallback += (sender, certificate, chain, sslPolicyErrors) => {
-                System.Diagnostics.Debug.WriteLine("Callback Server Certificate: " + sslPolicyErrors);
+            //ServicePointManager.ServerCertificateValidationCallback += (sender, certificate, chain, sslPolicyErrors) => {
```

Comment:
Yeah, because it's there as an example.  Although it doesn't actually work without specifying the customSSL flag.